solr/core/src/java/org/apache/solr/search/FunctionQParser.java

```diff
@@ -398,7 +398,7 @@ protected ValueSource parseValueSource(int flags) throws SyntaxError {
     if ((ch >= '0' && ch <= '9') || ch == '.' || ch == '+' || ch == '-') {
       Number num = sp.getNumber();
       if (num instanceof Long) {
-        valueSource = new ValueSourceParser.LongConstValueSource(num.longValue());
+        valueSource = new ValueSourceParsers.LongConstValueSource(num.longValue());
```

Comment:
it appears inconsistent with the lower two lines; can you static import this?  Or move to top level if the below ones are (consistency)

Reply:
This one is actually interesting. The other two value sources are from Lucene, `LongConstValueSource` is from Solr, which is why this is inconsistent. I could add an import, but perhaps something else could be considered here?

Would `ConstValueSource` be the equivalent in Lucene for `long` values? Because I can only find `ConstValueSource` for `double` in Lucene.